Client/Frontend/Settings/SearchSettingsTableViewController.swift

```diff
@@ -177,9 +224,12 @@ class SearchSettingsTableViewController: ThemedTableViewController, FeatureFlagg
     }
 
     override func numberOfSections(in tableView: UITableView) -> Int {
-        guard featureFlags.isFeatureEnabled(.feltPrivacySimplifiedUI, checking: .buildOnly) else {
+        guard featureFlags.isFeatureEnabled(.firefoxSuggestFeature, checking: .buildAndUser) else {
             return Section.allCases.count - 1
         }
+        guard featureFlags.isFeatureEnabled(.feltPrivacySimplifiedUI, checking: .buildOnly) else {
+            return Section.allCases.count - 2
+        }
```

Comment:
I'm confused by this, I think lololol. This seems like we want to check both conditions, rather than just one, no? If `.firefoxSuggestionFeature` is false AND `.feltPrivacySimplifiedUI` is false, we want to return ` - 2` but here, we're returning ` - 1`

Reply:
oooh you're right, not sure why my brain got a little melty here